### PR TITLE
[FEAT]: added landing page

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created landing page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  opacity: 0;
+}
+.search__bar:focus:-ms-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
```
Comment (abhishektanwar) [inline index.css:41]: Rems could be used here . Any specific reason for using px here?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- main section -->
+    <main
+      class="relative w-full flex align-items-center justify-center"
+      style="top: 5rem; min-height: 100vh"
```
Comment (abhishektanwar) [inline index.html:99]: Please try to avoid inline styling.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    >
+      <div style="max-width: 980px; margin: auto">
```
Comment (abhishektanwar) [inline index.html:101]: Please try to avoid inline styling.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-cyan-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
```
Comment (abhishektanwar) [inline index.html:148]: It is a good practice to use rel="noopener noreferrer" with target="_blank" from security point of view.

**Review by abhishektanwar — APPROVED** [review]
Please check comments. Code is clean and is good to merge.

**Review by VedanthB — APPROVED** [review]
good work ananya,
please implement the changes mentions above, apart from that everything looks goods, 🚀

**Pull request merged**